### ISSUE TITLE
Monika hasn't asked what hemisphere I'm in, even when discussing auroas.

### ISSUE THREAD

**Issue by LeeSpork** [Monika-After-Story/MonikaModDev#9930]
I started playing the mod yesterday, and I heard somewhere that the mod can ask you what hemisphere you are in to set the seasons correctly. But since it is yet to do so, the mod inappropriately thinks it is Winter, even though I am in the Southern hemisphere.

Monika has even brought up the aurora topic a few times, in which she discusses its differences between the Northern and Southern hemispheres. I find it kind of jarring and frustrating that despite the hemispheres (as well as Winter) being on her mind in this chat, she hasn't thought to ask me which one I'm in.

I'm not sure if this counts as a suggestion or a bug.
One way to remedy this would be to have her ask about your hemisphere during that topic, if she hasn't already.
Another, more drastic solution could be to have seasons disabled entirely and not marked on the calendar until Monika knows which hemisphere you are in. (When does she ask, btw?)

**Comment by Iceninja64** [comment]
1 she probably already asked you what hemisphere you already lived in. You can find the conversation if you look up hemisphere. 2 you haven't unlocked the conversation yet and leaving Mas open in the background can be a option to get that conversation

**Comment by LeeSpork** [comment]
> 1 she probably already asked you what hemisphere you already lived in. You can find the conversation if you look up hemisphere.

She hasn't

> 2 you haven't unlocked the conversation yet and leaving Mas open in the background can be a option to get that conversation

Having be completely random feels kind of silly, considering she is left just incorrectly assuming I have Northern Hemisphere weather, with no way for me to bring is up myself. I think this reinforces my suggestion.

Maybe another way to fix my issue could be to have the conversations on weather events like the auroras be made to not come up unless she has asked about my hemisphere?

**Comment by Booplicate** [comment]
Auroras can happen in either hemisphere regardless of where you live.

**Comment by LeeSpork** [comment]
Yes, but she said it they only happen during Winter, unless I'm mistaken

**Comment by LeeSpork** [comment]
Update: I've been talking to Monika daily for 2 weeks now, and she is still not thinking to ask me what hemisphere I'm in, even while asking me if it ever snows where I live.

**Comment by Booplicate** [comment]
The topics you've mention here are not related to hemispheres directly, you can get them before talking about hemispheres. The topic about hemispheres is available from the start, but it's random.

**Comment by LeeSpork** [comment]
Yes, but I think it is unthoughtful of Monika to be continuously randomly prioritizing Winter topics over allowing me tell her that it isn't Winter and I don't want it to be Winter

**Comment by Booplicate** [comment]
Valid, we should probably prioritise some topics more than the others, especially at the start.
Plus, we could push `monika_hemispheres` early and make other weather topics dependent on it.
